ext/semian/resource.h

```diff
@@ -0,0 +1,75 @@
+/*
+For core semian resource functions exposed directly to ruby.
+
+Functions here are associated with rubyland operations.
+*/
+
+#ifndef SEMIAN_RESOURCE_H
+#define SEMIAN_RESOURCE_H
+
+#include "semian.h" // FIXME remove this once temporary declarations are removed
+#include "types.h"
+
+// Ruby variables
+ID id_timeout;
```

Comment:
@csfrancis note that this must be in a header because both semian.c and resource.c need to access it.